pkg/app/piped/notifier/slack.go

```diff
@@ -271,3 +267,15 @@ func makeSlackMessage(title, titleLink, text, color string, timestamp int64, fie
 		}},
 	}
 }
+
+func addAccountsToText(text string, accounts []string) string {
+	if len(accounts) > 0 {
+		approvers := make([]string, 0, len(accounts))
+		for _, a := range accounts {
+			approvers = append(approvers, fmt.Sprintf("<@%s>", a))
+		}
+		Newtext := fmt.Sprintf("%s %s", strings.Join(approvers, " "), text)
+		return Newtext
+	}
+	return text
```

Comment:
nit: Basically, we value the early return as much as possible.
```suggestion
	if len(accounts) == 0 {
	    return text
	}
	approvers := make([]string, 0, len(accounts))
	for _, a := range accounts {
		approvers = append(approvers, fmt.Sprintf("<@%s>", a))
	}
	Newtext := fmt.Sprintf("%s %s", strings.Join(approvers, " "), text)
	return Newtext
}
```

Reply:
Fundamentally, deeply nested code gets difficult to understand. Besides, once it returns, we don't have to follow any of the following codes.

Reply:
That makes sense to me. Thanks, I'll update it.